README.md

```diff
@@ -1,3 +1,32 @@
 # adapt-navigationTitle
 
-AT Compatible
+**Navigation Title** is an *extension* for displaying a title in the course navigation bar. The title can be hidden for mobile view.
+
+<img src='https://user-images.githubusercontent.com/898168/210417005-3c2f0e9d-b1f0-4a7b-815c-33c3a6921965.jpg' width="500" alt="Screenshot">
+
+### Attributes
+
+The following attributes are set within *course.json*.
+
+**\_navigationTitle** (object): The course.json Navigation Title target attribute object.
+
+>**\_isEnabled** (boolean): Controls whether the Navigation Title course object is enabled or not.
```

Comment:
Could the format of the readme entries follow the newer format please?

https://github.com/cgkineo/adapt-footer/pull/8/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R15

Reply:
Should we include "AAT support" in the information at the bottom (ex. Version number, Author / maintainer, etc)?